.versionrc

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This file could be removed. Keep it here just in case we want to bump the monorepo version number